app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+    <string name="confirm_dir_exclude_included">“%1$s” wordt al ingesloten. Wilt u het uitsluiten?</string>
+    <string name="confirm_dir_already_excluded">“%1$s” wordt al uitgesloten</string>
+    <string name="confirm_dir_already_included">“%1$s” wordt al ingesloten</string>
+    <string name="message_removed_directory">Map “%1$s” is verwijderd</string>
+    <string name="nothing_playing">Er wordt niets afgespeeld</string>
+    <string name="unknown_artist">Onbekende Artiest</string>
+    <string name="unknown_album">Onbekend Album</string>
+    <string name="unknown">Onbekend</string>
+    <string name="empty_no_permission">Jockey heeft rechten nodig om uw muziekbibliotheek uit te lezen</string>
+    <string name="empty_no_permission_detail">Om uw muziek te laten zien heeft Jockey rechten nodig tot de externe opslag</string>
+    <string name="error_no_permission_dirs">Om mappen uit te sluiten heeft Jockey toegang nodig tot de externe opslag</string>
+    <string name="empty">Hier is geen muziek te vinden</string>
+    <string name="empty_detail">Je kan muziek synchroniseren vanaf de computer of downloaden vanaf het internet</string>
+    <string name="empty_genres">Kan geen genres vinden</string>
+    <string name="empty_playlists">Je hebt nog geen afspeellijsten</string>
+    <string name="empty_playlist">Deze afspeellijst is leeg</string>
+    <string name="empty_auto_playlist">Deze afspeellijst is leeg</string>
+    <string name="empty_search">Geen resultaten</string>
+    <string name="empty_auto_playlist_detail">Geen nummers die je filter volgden</string>
+    <string name="empty_playlist_detail">Je kan muziek toevoegen in de bibliotheek pagina</string>
+    <string name="empty_playlists_detail">Je kan een nieuwe maken door op de “+” knop te drukken</string>
+    <string name="empty_genres_detail">Zorg ervoor dat je muziek correct is getagd</string>
+    <string name="empty_queue">Er wordt niets afgespeeld</string>
+    <string name="empty_error_album">Kan het album niet laden</string>
+    <string name="empty_error_artist">Kan de artiest niet laden</string>
+    <string name="empty_error_genre">Kan het genre niet laden</string>
+    <string name="empty_error_folder">Kan de map niet laden</string>
+    <string name="color_grey">Grijs</string>
+    <string name="color_red">Rood</string>
+    <string name="color_orange">Oranje</string>
+    <string name="color_yellow">Geel</string>
+    <string name="color_green">Groen</string>
+    <string name="color_cyan">Cyaan</string>
+    <string name="color_blue">Blauw</string>
+    <string name="color_purple">Paars</string>
+    <string name="color_black">Zwart</string>
+    <string name="color_teal">Teal</string>
+    <string name="theme_dark">Donker</string>
+    <string name="theme_light">Licht</string>
+    <string name="theme_auto">Automatisch</string>
+    <string name="playlist_maximum_prefix">Sta niet meer toe dan</string>
+    <string name="playlist_maximum_suffix">nummers</string>
```

Comment:
Looks like "Allow no more than 25 songs" gets literally translated as "Allow no more than 25 numbers"? Is that correct here?

Reply:
The correct translation of "Allow no more than 25 songs" is "Sta niet meer toe dan 25 nummers".

I do not see any problem here?